src/utils/columnUtils.js

```diff
@@ -1,23 +1,14 @@
-function containsColumnPropertiesArray(rowProperties) {
-  return (rowProperties &&
-    rowProperties.props &&
-    rowProperties.props.children &&
-    Array.isArray(rowProperties.props.children));
-}
-
-function containsColumnPropertiesObject(rowProperties) {
-  return rowProperties && rowProperties.props && rowProperties.props.children;
-}
-
 /** Gets a column properties object from an array of columnNames
  * @param {Array<string>} columns - array of column names
  */
-export function getColumnPropertiesFromColumnArray(columns) {
+function getColumnPropertiesFromColumnArray(columnProperties, columns) {
```

Comment:
Technically this is a breaking change, but there's really no reason for this to have been used publicly.